adminSiteServer/apiRoutes/gdocs.ts

```diff
@@ -282,6 +284,12 @@ export async function deleteGdoc(
                 [gdocSlug, "/"]
             )
         }
+        // Make sure the old WP post that was replaced by this GDoc doesn't
+        // become available again (if it exists). By this time we don't want
+        // that behavior by default (any more).
+        await trx<DbRawPost>(PostsTableName)
+            .where("slug", gdocSlug)
+            .update({ gdocSuccessorId: null, status: "private" })
```

Comment:
I'm never quite sure at this point, do we also need to update `wpApiSnapshot->"$.status"`?